drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/PriorityListCheck.java

```diff
@@ -42,18 +42,8 @@ public boolean isActive( final CheckConfiguration checkConfiguration ) {
     }
 
     @Override
-    public void check() {
-        checkWithIssues = null;
-
-        for ( final Check check : filteredSet ) {
-            check.check();
-            if ( check.hasIssues() ) {
-                checkWithIssues = check;
-            }
-
-            if ( checkWithIssues != null ) {
-                break;
-            }
-        }
+    public boolean check() {
+        checkWithIssues = filteredSet.stream().filter( Check::check ).findFirst().orElse( null );
```

Comment:
@jomarko anyMatch() returns a boolean, while findFirst() the first object found in the stream. They have different purposes.